src/yowsup_if.js

```diff
@@ -836,35 +837,7 @@ CoSeMe.namespace('yowsup.readerThread', (function() {
 
   var alive = false;
 
-  var _onPing;
-  var _ping;
-  var _autoPong;
-
   return {
-    set onPing(aFun) {
```

Comment:
this has been moved into the _signalInterface. BTW, maybe an empty function should have been set as the _onPing callback instead of null (then the check for null can be removed)